app/pages/lab/workflows.jsx

```diff
@@ -7,116 +7,161 @@ import WorkflowCreateForm from './workflow-create-form';
 import LoadingIndicator from '../../components/loading-indicator';
 import Paginator from '../../talk/lib/paginator';
 
-const WorkflowsPage = (props) => {
-  const renderWorkflowTable = ((workflow) => {
+class WorkflowsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleWorkflowSettingChange = this.handleWorkflowSettingChange.bind(this);
+    this.state = {
+      active: false,
+      error: null
+    };
+  }
+
+  handleWorkflowSettingChange(workflow, e) {
```

Comment:
I think the intention of this component was just to be a functional view component. API calls should go into `WorkflowsContainer`